base/applications/rapps/settingsdlg.cpp

```diff
@@ -94,18 +94,61 @@ IsUrlValid(const WCHAR *Url)
 
 namespace
 {
-inline BOOL
+static inline BOOL
 IsCheckedDlgItem(HWND hDlg, INT nIDDlgItem)
 {
-    return (SendDlgItemMessageW(hDlg, nIDDlgItem, BM_GETCHECK, 0, 0) == BST_CHECKED) ? TRUE : FALSE;
+    return SendDlgItemMessageW(hDlg, nIDDlgItem, BM_GETCHECK, 0, 0) == BST_CHECKED;
 }
 
-VOID
+static void
+HandleGeneralListItems(HWND hWndList, PSETTINGS_INFO Load, PSETTINGS_INFO Save)
+{
+    PSETTINGS_INFO Info = Load ? Load : Save;
+    const struct {
+        WORD Id;
+        BOOL *Member;
```

Comment:
```suggestion
        PBOOL Member;
```
And maybe "Member" -> "Setting".

Reply:
Allow me to quote from [Windows Coding Conventions](https://learn.microsoft.com/en-us/windows/win32/learnwin32/windows-coding-conventions):

 > The LP prefix has been preserved to make it easier to port 16-bit code to 32-bit Windows. Today there is no distinction, and these pointer types are all equivalent. **Avoid using these prefixes**; or if you must use one, then use P.

The ROS guidelines do not say anything about this.